frontend/model/contracts/group.js

```diff
@@ -1686,12 +1685,6 @@ sbp('chelonia/defineContract', {
       }
 
       if (memberID === identityContractID) {
-        // NOTE: remove all notifications whose scope is in this group
-        // TODO: FIND ANOTHER WAY OF DOING THIS WITHOUT ROOTGETTERS
-        for (const notification of sbp('state/vuex/getters').notificationsByGroup(contractID)) {
```

Comment:
This logic was moved to the `LEFT_GROUP` event handler.

Reply:
Please add a comment then here explaining that.